README.md

```diff
@@ -735,9 +735,10 @@ def call_agent(
 ```
 
 Agent wait function: Waits for creation of edge to reaction node for some seconds.
+Default reaction_node is `question_finished`.
 
 ```python
-def wait_agent(seconds: float, question_node: ScAddr, reaction_node: ScAddr) -> None: ...
+def wait_agent(seconds: float, question_node: ScAddr, reaction_node: ScAddr = None) -> None: ...
```

Comment:
In the description you say that default reaction_node is `question_finished`, but in the code you add None as the value of node by default

Reply:
Bottom